src/Microsoft.VisualStudio.Editors/PropPages/xlf/Strings.de.xlf

```diff
@@ -62,7 +62,7 @@
       </trans-unit>
       <trans-unit id="none">
         <source>none</source>
-        <target state="new">none</target>
+        <target state="translated">Keine</target>
```

Comment:
The strings above/below this one were not translated.

Reply:
@cristianosuzuki77 will you be able to investigate why these two strings are untranslated before this is merged?